sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateContainerQcDao.java

```diff
@@ -57,4 +58,15 @@ public long save(QC qc) throws IOException {
     }
   }
 
+  private SequencerPartitionContainer getSequencerPartitionContainer(long id) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)